disdrodb/data_transfer/download_data.py

```diff
@@ -58,20 +58,21 @@ def _download_file_from_url(url: str, dir_path: str, force: bool = False) -> str
     fname = os.path.basename(url)
     file_path = os.path.join(dir_path, fname)
 
-    if not force and os.path.isfile(file_path):
-        print(f"{file_path} already exists, skipping download.")
-        return file_path
-    elif force and os.path.isfile(file_path):
-        os.remove(file_path)
+    if os.path.isfile(file_path):
+        if force:
+            os.remove(file_path)
+        else:
+            print(f"{file_path} already exists, skipping download.")
+            return file_path
 
     downloader = pooch.HTTPDownloader(progressbar=True)
     pooch.retrieve(url=url, known_hash=None, path=dir_path, fname=fname, downloader=downloader, progressbar=tqdm)
 
     return file_path
 
 
-def _download_station_data(metadata_fpaths: str, force: bool = False) -> None:
-    """Download the station data.
+def _download_unzip_station_data(metadata_fpath: str, force: bool = False) -> None:

```

Comment:
I would prefer the previous function name ...

Reply:
ok